README

```diff
@@ -0,0 +1,73 @@
+====== hocr-tools ======
+
+Tools for manipulating and evaluating the hOCR microformat for
+representing multi-lingual OCR results.
+
+hOCR is a format for representing OCR output, including layout
```

Comment:
@kba This file was already replaced with the markdown file. Therefore this should be deleted from the PR. (Maybe check the commit https://github.com/tmbdev/hocr-tools/pull/10/commits/91daa590d9661d85da61d6242752fb1bc27f19f1 if there are any other unwanted side-effects.)